### PR TITLE
Unwrap common, core, ext and frontend

### PR DESCRIPTION
**Pull request by bikallem**

### CODE REVIEW

File: jscomp/bsb/bsb_build_util.ml [hunk 1]
```diff
@@ -22,6 +22,8 @@
  * along with this program; if not, write to the Free Software
  * Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA 02111-1307, USA. *)
 
+open! Ext
```
Comment (EduardoRFS) [inline jscomp/bsb/bsb_build_util.ml:25]: Why `open!`?

Comment (bikallem) [reply jscomp/bsb/bsb_build_util.ml:25]: https://stackoverflow.com/questions/22607755/what-does-open-mean-in-ocaml

**Comment by anmonteiro** [comment]
Hey, thanks for spending time on this. The GitHub PR process is, however, a little backwards, and I should've provided more context on the issue.

I'm not a big fan of this for a few reasons:

- the diff is super large, and it seems really invasive. Cherry picking upstream patches will now have even more merge conflicts
- for `Ext`, given that most modules are already prefixed with `Ext_`, using a wrapper library is just cumbersome
- It's not clear to me that `core` and `frontend` should be the names of those libraries. `Core`, especially, seems very confusing.

I think we'll probably move in this direction, but I'd prefer to start with a diff that is way more minimal.

**Comment by bikallem** [comment]
All right. I will close this an probably send smaller PRs.

**Pull request closed without merging**